x-pack/plugins/lens/server/routes/index.ts

```diff
@@ -8,9 +8,7 @@
 import { CoreSetup, Logger } from '@kbn/core/server';
 import { PluginStartContract } from '../plugin';
 import { existingFieldsRoute } from './existing_fields';
-import { initFieldsRoute } from './field_stats';
 
 export function setupRoutes(setup: CoreSetup<PluginStartContract>, logger: Logger) {
   existingFieldsRoute(setup, logger);
-  initFieldsRoute(setup);
```

Comment:
Should we add `release_note:plugin_api_changes` PR label? How can we document this breaking change?

Reply:
Lens server API for fetching field stats was refactored into UnifiedFieldList public service and server API.

Reply:
No need to publish this is API change, it's an internal undocumented API